src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -737,6 +740,9 @@ def load_arguments(self, _):
                    help='The container registry server password. Required for private registries.')
         c.argument('min_replicas', type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
         c.argument('max_replicas', type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
+        c.argument('workload_profile_name', help="The name of the workload profile to run the app on.", is_preview=True)
+        c.argument('cpu', type=int, help="Required CPU in cores from 0.5 to 2.0.", is_preview=True)
+        c.argument('memory', help="Required momory from 1.0 to 4.0 ending with ""Gi"" e.g. 1.0Gi, ", is_preview=True)
```

Comment:
This is missing `type=int`

Reply:
Memory is string. User is supposed to include the "Gi" prefix.

Reply:
Got cha. In that case, make sure L402 in this file does not say `type=int`

Reply:
Yeah! I noticed it after your comment. I am removing it from there too.